### PR TITLE
Add vertical heat flux visualization postprocessor

### PR DESCRIPTION
**Pull request by ian-r-rose**
Similar to #569, but a visualization postprocessor.

### CODE REVIEW

File: source/postprocess/visualization/vertical_heat_flux.cc [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+{
+  namespace Postprocess
+  {
+    namespace VisualizationPostprocessors
+    {
+      ASPECT_REGISTER_VISUALIZATION_POSTPROCESSOR(VerticalHeatFlux,
+                                                  "vertical heat flux",
+                                                  "A visualization output object that generates output "
+                                                  "for the heat flux in the vertical direction, with "
+                                                  "the sign convention of positive flux upwards.")
```
Comment (bangerth) [inline source/postprocess/visualization/vertical_heat_flux.cc]: Can you clarify this by saying that it is the *total* heat flux, i.e., conductive plus convective?

**Comment by bangerth** [comment]
Nice work. Please just fix the one comment string.

**Comment by bangerth** [comment]
Please rebase to master (you'll likely have to adjust the changes.h file to resolve the merge conflict) and I will merge. Thanks for writing this!

**Comment by ian-r-rose** [comment]
Okay, done.

**Pull request merged**